dashboard/app/views/levels/_multi.html.haml

```diff
@@ -227,6 +278,48 @@
     }
   });
 
+  $('.unsubmitButton').click(function() {
+
+    var dialog = new Dialog({
+      body:
+        '<div class="modal-content no-modal-icon">' +
+          '<p class="dialog-title">Unsubmit answer</p>' +
+          '<p class="dialog-body">' +
+          'This will unsubmit your last answer.' +
+          '</p>' +
+          '<button id="continue-button">Okay</button>' +
+          '<button id="cancel-button">Cancel</button>' +
+        '</div>'
+    });
+
+    var dialog_div = $(dialog.div);
+    dialog.show();
+
+    dialog_div.find('#continue-button').click(function () {
+      $.post(window.appOptions.unsubmitUrl,
+        {"_method": 'PUT', user_level: {best_result: 1, submitted: false}},
+        function(data)
+        {
+          location.reload();
+        }
+      );
+    });
+
+    dialog_div.find('#cancel-button').click(function () {
+      dialog.hide();
+    });
+
+    return;
```

Comment:
is this return here deliberately so that the code below isn't run or was this left in by mistake?